src/repos/NotificationRepo.ts

```diff
@@ -101,10 +101,91 @@ const sendNewArticleAndroidNotification = async (
     });
 };
 
+/**
+ * Send IOS push notification to {userID} that their weekly debrief is ready.
+ */
+const sendWeeklyDebriefIOSNotification = async (user: User): Promise<void> => {
+  // Get device token of user
+  const { deviceToken } = user;
+
+  // Setup IOS Admin
+  const apnProvider = new apn.Provider({
+    token: {
+      key: process.env.APNS_TOKEN_PATH,
+      keyId: process.env.APNS_KEY_ID,
+      teamId: process.env.APNS_TEAM_ID,
+    },
+  });
+
+  const notifTitle = 'Your Weekly Debrief is Ready';
+  const notifBody = 'Click to check out whats new in Volume this week!';
+
+  const note = new apn.Notification({
+    alert: {
+      title: notifTitle,
+      body: notifBody,
+    },
+  });
+
+  note.expiry = Math.floor(Date.now() / 1000) + 3600;
+  note.badge = 3;
+  note.sound = 'ping.aiff';
+  note.payload = {
+    messageFrom: 'Volume',
+    userID: user.uuid,
+    notifcationType: 'weekly_debrief',
+  };
+  note.topic = process.env.APNS_BUNDLE_ID;
+
+  // Send notification to APN servers
+  apnProvider.send(note, deviceToken).then((result) => {
+    console.log(result.sent);
+    console.log(result.failed);
+  });
+};
+
+/**
+ * Send Android push notification to {userID} that their weekly debrief is ready.
+ */
+const sendWeeklyDebriefAndroidNotification = async (user: User): Promise<void> => {
+  // Get device token of user
+  const { deviceToken } = user;
+
+  const notifTitle = 'Your Weekly Debrief is Ready';
+  const notifBody = 'Click to check out whats new in Volume this week!';
```

Comment:
what's

Reply:
these constants are the same as the one in the Android function, make sure to factor these out somewhere else (constants file?)